include/dgl/aten/coo.h

```diff
@@ -56,14 +56,16 @@ struct COOMatrix {
   /** @brief constructor */
   COOMatrix(
```

Comment:
@nv-dlasalle   Thanks for the suggestion. I updated this part accordingly. 
@frozenbugs  No, this is not checked in `CheckValidity`. Currently, all three arrays need to be pinned, otherwise, new pinned COO/CSR matrix will be allocated.